variablegroups/RPE-AKS-Common.json

```diff
@@ -22,7 +22,7 @@
       "value": "10.10.4.0/24"
```

Comment:
this cluster hasn't been used in forever, not sure it even exists, might be good to just delete it so you don't have to keep updating it?